packages/ocaml-config/ocaml-config.1/opam

```diff
@@ -14,7 +14,7 @@ depends: [
 ]
 substs: "gen_ocaml_config.ml"
 extra-files: [
-  ["gen_ocaml_config.ml.in" "md5=71f88c92be191cf4a0547f3c6196ef00"]
-  ["ocaml-config.install" "md5=8e50c5e2517d3463b3aad649748cafd7"]
+  ["gen_ocaml_config.ml.in" "md5=efb951a1b7bb6040b99dcb59267e1b56"]
```

Comment:
The hash doesn't match apparently:
```
#=== ERROR while compiling ocaml-config.1 =====================================#
Bad hash for /home/opam/.opam/repo/default/packages/ocaml-config/ocaml-config.1/files/gen_ocaml_config.ml.in
```

Reply:
Indeed. I accidentally changed the wrong checksum. Fixed and tested.